roles/fs-drift/tasks/main.yml

```diff
@@ -30,11 +30,11 @@
 
   - set_fact:
       fs_drift_path: /tmp
-    when: fs_drift.worker_pods > 0 and fs_drift.storageclass is not defined
+    when: workload_args.worker_pods|default('1') > 0 and workload_args.storageclass is not defined
```

Comment:
@rsevilla87 I tested this without worker_pods being defined and it did not have any issue.

Reply:
You're right, it doesn't fail in this version of the ansible-operator since it's using python2.7

```
Python 2.7.5 (default, Oct 30 2018, 23:45:53) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> '1' > 0
```

and python3
```
Python 3.8.2 (default, Feb 28 2020, 00:00:00) 
[GCC 10.0.1 20200216 (Red Hat 10.0.1-0.8)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> '1' > 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '>' not supported between instances of 'str' and 'int'
```

So I believe it will fail as soon as we migrate to a new version of the ansible-operator based in python3.

Reply:
Ahhhhh ok gotcha. I updated to cast them as int's.